pygmt/tests/test_config.py

```diff
@@ -23,75 +22,53 @@ def test_config():
             region="0/10/0/10",
             projection="X10c/10c",
```

Comment:
This test (test_config) seems to be one of the top 5 slowest tests, taking 2.47s at https://github.com/GenericMappingTools/pygmt/pull/1165/checks?check_run_id=2258162018#step:11:876 (though I don't see why it's so slow since it's just plotting basemaps). Could we simplify the test somehow, e.g. to plot a smaller frame and use a different config setting perhaps?

Reply:
Speed up the test by plotting a smaller basemap (10cX10c to 5cX5c).

Reply:
Ok, 1.43s now at https://github.com/GenericMappingTools/pygmt/pull/1165/checks?check_run_id=2262523142#step:11:820. Much better!